python/plugins/db_manager/db_plugins/data_model.py

```diff
@@ -139,6 +148,43 @@ def rowCount(self, index=None):
         return self.table.rowCount if self.table.rowCount is not None and self.columnCount(index) > 0 else 0
 
 
+class SqlResultModelAsync(QObject):
+
+    done = pyqtSignal()
+
+    def __init__(self):
+        QObject.__init__(self)
```

Comment:
can we switch to using python3 super() calls by convention? they are much nicer

Reply:
Oh silly me... Done.